tests/e2e/scenarios/test_admin_api.py

```
@@ -132,6 +135,73 @@ async def test_get_user_detail(admin_client, test_user):
     assert user["display_name"] == "E2E Test User"
 
 
+async def test_admin_token_visibility_matches_user_creation_lifecycle(
+    admin_client, reborn_v2_page, reborn_v2_server, test_user
```

Ruff (0.15.21)

[warning] Unused function argument: `test_user`

(ARG001)